docs/syndesis-operator-states.xml

```diff
@@ -0,0 +1 @@
ce"><diagram id="8ecc18c7-d5c6-3091-a06c-258d5145b2f6" name="Page-1">7V1Zs6JGFP4193EsemN5nLmzJFVJKpVbqSSPqO2SQdogzr3m16eRbrAXAVnUifgwI03TIOc75zsb3Cf0vHn7koTb1c9sTqMn6MzfntDHJwgBhi7/Lxs55CMeAvnAMlnPxaRy4GX9LxWDjhjdr+d0p0xMGYvS9VYdnLE4prNUGQuThL2q0xYsUs+6DZfUGHiZhZE5+sd6nq7yUR965fgPdL1cyTMDN8j3TMPZ12XC9rE43xNEi+Mn370J5Vrih+5W4Zy9ngyhT0/oOWEszb9t3p5plN1bedvy4z6f2Vtcd0LjtMkBMD/gWxjtqbzi43WlB3kv+AH8tvOND6t0E/ExwL/y695m+3dpmKQvaZhm+xfrKHpmEUuOByLn+Mkmpwn7Sk/2LBZiTxROafShuGdySszi43osTgU2OKDybdv65o8W9+EbTVL6djIkbsIXyjY0TQ58ypsEaH7EQSKY5NuvpfSRENnqRPByLBR4WxYLl/ecfxG3/bjJpn9niIXO8bef3nMpE3RypUJcp+KYc6SKTZakK7ZkcRh9KkdPxZTdgDWH9ftovYz52JSlKdvwHTSev8/0JFtkS+N8RNxrv4vIVAhAx4Mf/AtEySWYHP7kg8XGX9nGhFSJecf2yYwqgOawXFIxCwtjlN25SiwkNArT9TfVBthEKw79la3jtMQQ9D0VRDo68osSR2kAKS7Dhhm+KWBj12Js0WI3SsV9VvDj/rNncse73VEi7/kECLdv5U7+bZn9/2PM9Zurf7yU6/EryZfMJxjG4ogJOhfwe12tU/qyDY/CeeVUoaIzTGYCE9gxsSPspgWLBVruxnwAR7UfCJn2AxBiGhD3cgNiSN/tLH3g2qT/OaG7Fd8rUNAYAiu2me539eL/juUNMZgQReLQwhjARhlBDxL3DYkfuLdkEPclTBEKhojoIrUQR8q2t2UN0Iw1jkRR8kZT1nAtrNGQNAQi3jkT7EjBCEy8k/LvSiwEKuv6kDQiFmMh7cxssdhR5Qgu3fBwMmObHb87f2XAB6oWBIq7yb/kKzbkOQPowUDE9pK5rSOt1dKajMeEeAm0GblgGFqT566MTf7XznAj0xWYpguQ6zi8wFXtkoea2aUWhgCgRwcDfVunRWDEv5dxUeeYCZsQCq6EIF9FEHFaMhvSLJWrL5T/XmOhSwnPU0+DXFJ5Wdp0QpTpnekRDBX4Hflxv/0criPOejcgSX9GZzObtk19gsl9kaSjWUEJRSX28wYiSWIg4JltthFNM7ndYWRW6cf3IA0ENItikwZyTWn0EZcBMxSPWTeeqonL7oG2qkMzSVs5VQniAqe0VWxcSFsS+4rrc2HY5kxgoEdthdfbkdwQ1sI20IzcLuUk5Kss4+GKIKw8+nwU2IaJPAP5Y0riXEpC6gQwXLnWuJfzGqcrQD8QJ66WB3PdYUCOg2BQV6pgmoFy6Ddxo+Yh9RdWN8qd+XS6uCc3CkNVwJ5nI+6Bcg3IlmvoQ/rv0zScrcZMU630kcqVhVSvUUBBpu4P7LbdOXed5afOqQZLol3q3gWZds/1sAKXnhLtINDCe79lOgIGnnZ9GkzPpCMG8s6kco3Zs4owpG3tSGLvJukzLb8EmnUc1AIf42HSZ0RL02G9PanmB2rzO3t9eEwrWxXjfCsOaBylWLLK1ypMFCltqRcNCxMmXmWmqOjraGbHu9ZRXVStF1DTi0vnE8/vV49siejH0qOKbFapY56uZI1UqWiBbRPwd1QlF6qdTa7ucDdVJU9Lk7t6wrUvikHqBXs1FCNVp+183+mZksyMfsyy8x+EGzimkiuoSmjYxU6cLafW1IkrUsnAk8mvrvzlarkQ0lLpkObXubqD2JPS6SloHNTwEcBV86+SssZD9U1+3G8jrlgpHTsmTzwcV0UIAFfsl8RmdWJM8BiWtJdoGFsyPE3778sEj+F2wH4Mq94mAnFLw4qBWva4cYIHjx3BdoQPkMK0IRxf6CtwiAcIaAjqBeGEaHa2rb8OfTy5L4wP1Qz8C0tvWaT7nnqdkKc/91Bw9jUKNRLbY5qhqq3AjI+a2TVLJ/G1Hp0DUMtPNewiMM2fqwYWxjN4PQU8GGsRWk28A0nF9M45AzJU88Lv22USzsfy9aXP/wWOzSpaug57sYrw4a3icPGMrYxBrpR7xb7eXRX4bRNB0CBuIDtlB65l+KRnYzfW7Bo8CtI2EWqrNVwL7yAAE+CWH63JI3CCiXu6u50qgMCZoJMPVtt/Pa/yLD0pCYTDVhHIUE+G5B4BzZ4M2Sc0gzjLXm6SLz1NSsdg9Bcq291UW+xY2t2gN1QUZVaYHtSAVmWHNAPauO+BWAxo4xa3rg6Dp7adBW2TP/pCHrmGUXS1alVQ09Wgz6+rOhHN1FdWndoYXVtF6bEU6+L0RCe9kiw3uGOi+wRakgFAojouLT0TrToRBO4geqZ76MDB1YpTd0B3zTELdI+mOYV2wGaNeBdULMgNY1itMaB1AFsXG7jXICigPVwKHFCtONjpNF8jwO5qZpYJe48KyhrKbaOB76iqgoEOE9vTL95Aj5ATW13tsWxvhYHt+Kym1fT61zG9yHOqTCZwfKcPr6XmNEH1WfpqQAt0FaqJBeoO6GxrpWo+/KtV9SjN8lxnT69WNWUwVF3sZ5osx6pYreShpxcdPEvX32DvxYQjsQ0WjksBKX1+V4opINJQ1fZ9iJioVXygtwP29tYoe9PM+WclKud356axhNbocZ12mnHDBLAOHNT2wTd9IcMgX6wZHTr/XLNgwf2ieB4mPNZ0nn/L/kkov28sNmDMqSrV/CsFchY8iSHZ/zrjwqaJBeSb9Xx+VAQboZeUf1+vx1ILT76Fjh1bnKnBqBUdm9lxkTbggz/RZTjLrlN0Yo7CbCBM4qvSJHAynDz5ZvlnKXIlLv/2B/r0Hw==</diagram></mxfile>
```

Comment:
There is a connection "Attaching" --> "Not Installed" missing in case of a parallel installation.